ui/src/main/webapp/package.json

```diff
@@ -114,7 +129,7 @@
     "typescript": "2.1.1",
     "typings": "1.4.0",
     "url-loader": "^0.5.6",
-    "webpack": "2.1.0-beta.25",
-    "webpack-dev-server": "2.1.0-beta.9"
+    "webpack": "2.1.0-beta.27",
+    "webpack-dev-server": "2.1.0-beta.11"
```

Comment:
Just a comment for the future - please next time create separate PRs for updating dependencies.